server/service/diff.go

```diff
@@ -24,3 +40,11 @@ func (d *Diff) Generate(nameA, nameB, contentA, contentB string) (string, error)
 
 	return difflib.GetUnifiedDiffString(diff)
 }
+
+// ReplaceInvisible preprocessor function that replace invisible character with visible onces
+func ReplaceInvisible(content string) string {
+	content = strings.Replace(content, " ", "·", -1)
+	content = strings.Replace(content, "\t", "→", -1)
```

Comment:
How about replacing tab with '→   ' to try to keep indentation? I think a single char indentation may make code less readable.

Reply:
I'm used to seeing only one &rarr; symbol.
If you want more space, we could use &#10230; (`&#10230;` or `&#x27f6;`).
but whatever you prefer.

Reply:
That's odd, I kind of expect any editor to show the symbol _and_ indent. Like VS Code for example:
![tabs](https://user-images.githubusercontent.com/1469173/37345744-e09131ec-26cd-11e8-9964-e0aea9bfc4fd.png)

Reply:
I definitely saw *only* → many times.